src/Dashboard.php

```diff
@@ -54,7 +54,7 @@ public function assets(): HtmlString
         }
 
         foreach ($this->inlineScripts as $inlineScript) {
-            $assets[] = "<script>{$inlineScript}</script>";
+            $assets[] = "<script>$inlineScript</script>";
         }
 
```

Comment:
You may need to remove The curly braces on the `{$stylesheets}`  also. Don't you think?